packages/backend/src/plugins/techdocs.ts

```diff
@@ -50,7 +50,7 @@ export default async function createPlugin({
   const urlPreparer = new UrlPreparer(reader, logger);
   preparers.register('url', urlPreparer);
 
-  const publisher = new LocalPublish(logger);
+  const publisher = new LocalPublish(logger, config);
```

Comment:
You want to inject `discovery` here instead of config, and look up `"techdocs"` using that!

Reply:
Ah! Thanks.

It does make sense to use `discovery.getBaseUrl("techdocs")` to get the baseUrl for TechDocs. :)